decentraland/unity-renderer
# Feat: Render Objects Using a Single Camera

## What does this PR change?



This PR removes the skybox camera and renders every 3D object using a single camera. The default lit shader and the skybox elements shaders were modified so the skybox elements are always rendered behind.

## How to test the changes?



1. Go to: https://play.decentraland.zone/?renderer-branch=feat/Render-Using-Single-Camera
2. Explore the world and verify everything is being render properly

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
